public/assets/js/main.js

```diff
@@ -49,3 +49,15 @@ function createCard (lake) {
 }
 
 getLakes()
+
+window.onscroll = function() {scrollFunction()};
+
+function scrollFunction() {
+  if (document.body.scrollTop > 80 || document.documentElement.scrollTop > 80) {
+    document.querySelector('.header-box').classList.add('smaller');
```

Comment:
assign it to a variable e.g. `const headerBox = document.que....`, cause you also use it later + makes code a bit more readable at a glance